test/test_jit.py

```diff
@@ -4062,6 +4062,18 @@ def forward(self, x, w):
         with self.assertRaisesRegex(RuntimeError, 'test_jit.py\", line {}'.format(lineno + 3)):
             loaded(torch.rand(3, 4), torch.rand(30, 40))
 
+    def test_ignorable_args(self):
+        def fn():
+            val = torch.tensor([[1, 2, 3, 4, 9, 9], [1, 2, 3, 4, 9, 9]])
+            return val[2::, :1:]
+        scripted = torch.jit.script(fn)
+        src = str(scripted.code)
+        # we ignore trailing arguments after start=2 for dim 0
```

Comment:
Optional:
It would be good if we include the schema of `slice` here to help with readability.

Reply:
sounds good